src/game/player.py

```diff
@@ -35,11 +35,19 @@ def reset(self):
     def is_alive(self):
         return self.hp > 0
 
+    def get_click_value(self):
+        item_click_value = sum(item.increase if item.bought else 0 for item in self.upgrades['Click'].values())
```

Comment:
We talked about this before. I think this is the right way for now, but you'll want to think of a way to optimize/cache it in the future as this will likely become a bottleneck. At least keep it in mind if things become too slow